### ISSUE TITLE
Overly generic css selector on menu

### ISSUE THREAD

**Issue by sunz7** [atom/find-and-replace#1032]
*Edit by @rsese to use the issue template*

### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

https://github.com/atom/find-and-replace/blob/befebde9a6830e126a156cb33b5a8b99a77191d6/menus/find-and-replace.cson#L31-L33

[A recent change](https://github.com/atom/find-and-replace/commit/befebde9a6830e126a156cb33b5a8b99a77191d6) made this selector more generic which causes an unintended menu item to be added.
We use .list-item in things like "File Tree". This overly generic css selector add "Copy Path" command to file tree's Context Menu. When user clicks on it, expecting it to copy the file path on for a folder or file, instead it does not. Please consider add more specificity to the selector.
Thanks!

### Steps to Reproduce

Using atom/atom:

1. `atom --safe .`
2. Right click on a tree view item

**Expected behavior:** [What you expect to happen]

No `Copy Path` item.

**Actual behavior:** [What actually happens]

A `Copy Path` item.

![copy-path](https://github-slack.s3.amazonaws.com/monosnap/Project__githubatom_2018-07-16_16-30-59.png)

**Reproduces how often:** [What percentage of the time does it reproduce?]

100%

### Versions

```
atom -v
Atom    : 1.30.0-dev-8209eaa63
Electron: 2.0.4
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

macOS 10.12.6

**Comment by rsese** [comment]
Thanks for the report!  I can reproduce with 1.30.0-dev-8209eaa63 on macOS 10.12.6.  I went ahead and updated the issue body to use our issue template, if you could use the issue template for future issues, that would be super helpful.

**Comment by sunz7** [comment]
@rsese oh sorry about that. I will use the template next time! Thank you!

**Comment by captbaritone** [comment]
When should we expect a version with this fix to get published? (Trying to decide if we should implement a work around)

**Comment by jasonrudolph** [comment]
@captbaritone: This fix will likely be available in [Atom Beta](https://atom.io/beta) within the next week. 

I'm under this impression that this bug is mostly an annoyance. If it's more than and annoyance, and this bug is having a significant negative impact on your use of Atom, please let us know. :bow:

**Comment by captbaritone** [comment]
This selector exists in the File Tree package in Nuclide. Nuclide users are seeing a broken "Copy Path" context menu item when right-clicking in the Nuclide file tree.

**Comment by jasonrudolph** [comment]
> This selector exists in the File Tree package in Nuclide. Nuclide users are seeing a broken "Copy Path" context menu item when right-clicking in the Nuclide file tree.

@captbaritone: Thanks for sharing that context. That is indeed more than an annoyance. 🙈 

I've cherry-picked this fix onto the `1.29-releases` branch (https://github.com/atom/atom/commit/e1d336f245a5dc0e6663848fd747814463089191), and it will be included in the Atom 1.29 stable release, which will likely ship this week.
